src/index.js

```diff
@@ -31,6 +33,8 @@ const enableQueryTracing = ENABLE_QUERY_TRACING === "true"
 const app = express()
 
 async function startApp() {
+  config.GRAVITY_XAPP_TOKEN = xapp.token
```

Comment:
... to here which seemed to fix the problem. Is this how you ended up doing it?

Reply:
Ah damn, I guess this means I have some unpushed patches locally. Yup, this is what I did too 👍